platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -126,6 +127,14 @@ void OfflineDatabase::migrateToVersion5() {
     db->exec("PRAGMA user_version = 5");
 }
 
+void OfflineDatabase::migrateToVersion6() {
```

Comment:
I'm a bit worried about this migration in combination with #9666. Any idea how long it will take if the database contains large offline regions?